### PR TITLE
Added base for header

### PR DESCRIPTION
**Pull request by slaveeks**
1. Added `HeaderLayout` layout
2. Added base styles for Header
3. Added `styles/colors.pcss`
4. Added Header to `App.vue`

### CODE REVIEW

File: src/App.vue [hunk 1]
```diff
@@ -1,15 +1,24 @@
...
+    <HeaderLayout />
     <router-view></router-view>
   </div>
 </template>
 
 <script lang="ts">
+import HeaderLayout from "./presentation/layout/HeaderLayout.vue";
 export default {
   name: "App",
+  components: {HeaderLayout},
```
Comment (kseniaSs) [inline src/App.vue:12]: It's question for @neSpecc, it is layout? So we have several layouts?

Comment (neSpecc) [reply src/App.vue:12]: No, it's just a component

File: src/App.vue [hunk 2]
```diff
@@ -1,15 +1,24 @@
...
 }
 </script>
 
-<style scoped>
-
-</style>
\ No newline at end of file
+<style>
+  * {
```
Comment (neSpecc) [inline src/App.vue:17]: use normalise.css instead

File: src/App.vue (hunk 1 above)
Comment (kseniaSs) [reply src/App.vue:12]: @slaveeks rename pls 🙏

Comment (slaveeks) [reply src/App.vue]: done

File: src/App.vue (hunk 2 above)
Comment (slaveeks) [reply src/App.vue:17]: As i understand, it is done

File: src/App.vue [hunk 3]
```diff
@@ -1,15 +1,25 @@
...
 
-<style scoped>
+<style>
+  * {
+    box-sizing: border-box;
+  }
 
-</style>
\ No newline at end of file
+  body {
```
Comment (kseniaSs) [inline src/App.vue:21]: After normalise, i think it's not neccassary

Comment (slaveeks) [reply src/App.vue]: Yes, removed

File: src/App.vue (hunk 2 above)
Comment (neSpecc) [reply src/App.vue:17]: so why does it still here?

File: src/App.vue [hunk 4]
```diff
@@ -1,15 +1,20 @@
...
+    <HeaderComponent />
     <router-view></router-view>
   </div>
 </template>
 
 <script lang="ts">
+import HeaderComponent from './presentation/components/HeaderComponent.vue';
 export default {
   name: "App",
+  components: {HeaderComponent},
```
Comment (neSpecc) [inline src/App.vue:12]: use script setup instead

File: src/App.vue [hunk 5]
```diff
@@ -1,15 +1,20 @@
 <template>
   <div id="app">
+    <HeaderComponent />
```
Comment (neSpecc) [inline src/App.vue:3]:
```suggestion
    <Header />
```

File: src/presentation/components/HeaderComponent.vue [hunk 6]
```diff
@@ -0,0 +1,21 @@
+<template>
+  <div class="header">
+  </div>
+</template>
+
+<script lang="ts">
```
Comment (neSpecc) [inline src/presentation/components/HeaderComponent.vue:6]: use script setup

File: src/presentation/components/HeaderComponent.vue [hunk 7]
```diff
@@ -0,0 +1,21 @@
...
+export default {
+  name: "HeaderComponent"
+}
+</script>
+
+<style scoped>
+  .header {
+    height: 52px;
```
Comment (neSpecc) [inline src/presentation/components/HeaderComponent.vue:14]: use variable

File: src/presentation/components/HeaderComponent.vue (hunk 6 above)
Comment (slaveeks) [reply src/presentation/components/HeaderComponent.vue]: done

File: src/App.vue (hunk 4 above)
Comment (slaveeks) [reply src/App.vue]: done

File: src/App.vue (hunk 2 above)
Comment (slaveeks) [reply src/App.vue:13]: about * {box-sizing: border-box} https://github.com/necolas/normalize.css/issues/202

File: src/App.vue [hunk 8]
```diff
@@ -1,15 +1,16 @@
...
     <router-view></router-view>
   </div>
 </template>
 
-<script lang="ts">
-export default {
-  name: "App",
-}
+<script lang="ts" setup>
+import Header from './presentation/components/HeaderComponent.vue';
```
Comment (neSpecc) [inline src/App.vue:9]:
```suggestion
import Header from './presentation/components/Header.vue';
```

**Review by neSpecc — APPROVED** [review]

**Review by kseniaSs — APPROVED** [review]

**Pull request merged**